src/test/encoding/types.h

```diff
@@ -192,6 +192,10 @@ TYPE_FEATUREFUL(InodeStore)
 TYPE_FEATUREFUL(MDSMap)
 TYPE_FEATUREFUL(MDSMap::mds_info_t)
 
+#include "mds/FSMap.h"
+//TYPE_FEATUREFUL(Filesystem)
```

Comment:
Ah, right. I was trying to decide if we'd ever want to decode a Filesystem separate from an FSMap, since it didn't come with all the functions required for this at the time I was doing debugging. Probably want to just drop it, but any thoughts? (And can you think of any other new types we're missing?)